goth/default-assets/docker/yagna-goth.Dockerfile

```diff
@@ -1,11 +1,15 @@
 # debian:bullseye-slim, 2021-05-12
 FROM debian@sha256:e827c9bc6913625ec75c8b466e6e79a6b936d0801956be607bce08a07078f57a
+
+RUN apt update \
+    && apt install -y libssl-dev ca-certificates \
+    && update-ca-certificates
```

Comment:
Do we run `update-ca-certificates` because we want to update them every time, or do we want to because we just installed `ca-certificates` package?
In second case it could be worth to create a tiny subproject to publish Docker image, lets say `debian-libssl-dev-ca-certs`  with these packages preinstalled so we could use it in this Dockerfile.
Also having `apt install`  might break test at some point even if we start `FROM` specific `debian` tag/hash version.

Reply:
It is not a blocker, just wanted to mention it.